ddpui/ddpairbyte/airbyte_service.py

```
@@ -888,68 +865,68 @@ def update_connection(
     if not isinstance(job_id, str):
         raise HttpError(400, "job_id must be a string")
 
-    res = abreq("jobs/get_without_logs", {"id": job_id})
+    res: Dict = abreq("jobs/get_without_logs", {"id": job_id})
     return res
 
 
 def get_jobs_for_connection(
-    connection_id: str, limit: int = 1, offset: int = 0, job_types: list[str] = ["sync"]
-) -> int | None:
+    connection_id: str, limit: int = 1, offset: int = 0, job_types: List[str] = ["sync"]
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)